app/renderer/actions/Session.js

```diff
@@ -410,15 +410,19 @@ export function newSession (caps, attachSessId = null) {
     dispatch({type: SESSION_LOADING});
 
 
-    const hostname = username && accessKey ? `${username}:${accessKey}@${host}` : host;
     const serverOpts = {
-      hostname,
+      hostname: host,
       port: parseInt(port, 10),
       protocol: https ? 'https' : 'http',
       path,
       connectionRetryCount: CONN_RETRIES,
     };
 
+    if (username && accessKey) {
```

Comment:
👍 
https://webdriver.io/docs/options/#user

Reply:
mm, these `user` and `key` is an issue...?
When I comment out here and sent a request with SauceLabs' tab, a different error returned.
So I guess this option did not work...(?)